src/components/CopyableInput/CopyableInput.scss

```diff
@@ -0,0 +1,5 @@
+.copyable-input {
```

Comment:
the class should be named something like `asab-copyable-input`

Reply:
okay